projects/igniteui-angular/src/lib/services/overlay/overlay.spec.ts

```diff
@@ -80,11 +73,11 @@ describe('igxOverlay', () => {
             imports: [IgxToggleModule, DynamicModule, NoopAnimationsModule],
             declarations: DIRECTIVE_COMPONENTS
         }).compileComponents();
-        clearOverlay();
+        HelperUtils.clearOverlay();
     });
 
-    afterAll(async () => {
-        clearOverlay();
+    afterAll(() => {
```

Comment:
Shouldn't we use "afterEach" instead of afterAll?

Reply:
There is no need, because in the beforeEach, 'clearOverlay' is called.